Description: fix new warnings caused by 50 patch.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2016-09-06
Index: wipe-0.22/wipe.c
===================================================================
--- wipe-0.22.orig/wipe.c
+++ wipe-0.22/wipe.c
@@ -93,6 +93,9 @@
 #include "misc.h"
 #include "version.h"
 
+// FIX warning: implicit declaration of function ‘dirname’ [-Wimplicit-function-declaration]
+#include <libgen.h>
+
 /* includes ***/
 
 /*** more defines */
@@ -610,6 +613,8 @@ static int wipe_filename_and_remove (cha
     int len;
     int r = -1;
     int fn_l, dn_l;
+    // FIX [-Wunused-but-set-variable]
+    (void)dn_l;
     char *oldname, *newname;
     char *dir, *dirc;
     dirc = strdup(fn);
