--- a/libs/models/gl_skin.c
+++ b/libs/models/gl_skin.c
@@ -185,6 +185,7 @@
 		s = &player_skin;
 	if (!s->tex)	// we haven't loaded the player model yet
 		return;
+
 	texnum = skin_textures + cmap;
 	fb_texnum = 0;
 	if (s->fb)
@@ -196,13 +197,13 @@
 	if (vid.is8bit) {
 		build_skin_8 (s->tex, texnum, skin_cmap[cmap],
 					  scaled_width, scaled_height, false);
-		if (s->fb)
+		if (s->fb && s->fb_tex)
 			build_skin_8 (s->fb_tex, fb_texnum, skin_cmap[cmap],
 						  scaled_width, scaled_height, true);
 	} else {
 		build_skin_32 (s->tex, texnum, skin_cmap[cmap],
 					   scaled_width, scaled_height, false);
-		if (s->fb)
+		if (s->fb && s->fb_tex)
 			build_skin_32 (s->fb_tex, fb_texnum, skin_cmap[cmap],
 						   scaled_width, scaled_height, true);
 	}
